resources/views/auth/register-form.blade.php

```diff
@@ -8,7 +8,7 @@ class="flex flex-col p-8 mx-4 border rounded-lg border-theme-secondary-200 md:mx
     @if($invitation)
         <input type="hidden" name="name" value="{{ $invitation->name }}" />
     @else
-        <div class="mb-4">
+        <div class="mb-8">
```

Comment:
instead of adding `mb-8` on each div, you might want to look if you can use `space-y-8` instead on the parent

Reply:
Doesn't look good cause then you have a big space between the top of the form and the first input field, could probably create a CSS class to not apply it on first child tho

Reply:
I see, csrf adds a hidden input at the top which triggers space-y-8 on the "first" element